src/app/modules/ix-forms/components/ix-input/ix-input.component.html

```diff
@@ -3,6 +3,7 @@
   [label]="label"
   [tooltip]="tooltip"
   [required]="required"
+  [class.disabled]="isDisabled"
```

Comment:
Disabled `ix-input` now looks different to every other component.
Either:
1. Introduce disabled input in ix-label and add support to other components.
2. Revert this change.